packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -104,11 +104,7 @@ export default defineComponent<{
 			type: String,
 		},
 		filter: {
-			default: (): KottiFilters.Filter => ({
```

Comment:
I think this change has an effect on all mentions of `props.filter.*` because it used to be null as default, for all 3 attributes (key, operation, value) but now it's not so fallbacks have to be put in place 

Can you run a check through the whole file and add it 

Alternatively, I can leave suggestions